installer/bundle/bundle_skel_Linux.sh

```diff
@@ -616,11 +616,9 @@ case "$installMode" in
     U)
         echo "Updating cross-platform agent ..."
         if [ $PROVIDER_ONLY -eq 0 ]; then
+	    shouldInstall_omi
+	    pkg_upd_list $OMI_PKG omi $?
             check_if_pkg_is_installed omi
```

Comment:
Huh? I don't think this is correct.

First, indentation is correct on 619-620. But more importantly, why are you checking if OMI should be installed?

When you look at removing 621, also look to see if check_if_pkg_is_installed is even used anymore. It may not be.

Finally, this is to fix an upgrade bug. What version of the older kit needs to be installed to trigger this? I'd like to verify, myself (regardless of when this is committed), that this problem is fixed.